File: server/PCMonitorServer/src/main/java/com/zogirdex/pcmonitorserver/MonitorDataDTO.java
package com.zogirdex.pcmonitorserver;

/**
 *
 * @author tom3k
 *
 * DTO — Data Transfer Object
 *
 * Dane diagnostyczne - przychodzące
 *
 *  * Nazwy parametrów tej klasy muszą być zgodne z danymi w json, można by też użyć adnotacji, np.: (w przypadku google.gson)
 * @SerializedName("HardwareName")
 */
public class MonitorDataDTO {
	public String HardwareName;
	public String SubHardwareName;
	public String SensorName;
	public String SensorType;
	public Float SensorValue;
	public String TimestampUtc;
}

File: client/PCMonitorClient/PCMonitorClient/MonitorDataDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PCMonitor
{
    // Dane pojedyńczego sensora
    public class MonitorDataDTO
    {
        public string HardwareName { get; set; }
        public string SubHardwareName { get; set; } // null jeśli brak
        public string SensorName { get; set; }
        public string SensorType { get; set; }
        public float? SensorValue { get; set; }
        public string TimestampUtc { get; set; }
    }
}
